code/game/machinery/syndicatebeacon.dm

```diff
@@ -125,6 +125,9 @@
 	for(var/obj/singularity/singulo in world)
 		if(singulo.z == z)
 			singulo.target = src
+	for(var/obj/singularity/energy_ball/tesla in world)

```

Comment:
This code isn't needed if robustin changes it to use target instead of a new var.